tests/audit-tests/configuration.properties

```diff
@@ -0,0 +1,20 @@
+browser=chrome 
```

Comment:
We can remove this - I'm waiting on Des to fix this so that we don't need to show the urls

I'll talk to Denny on Tuesday